pymongo/client_session.py

```diff
@@ -947,9 +947,18 @@ def _txn_read_preference(self):
             return self._transaction.opts.read_preference
         return None
 
+    def _start_serv_sess(self):
```

Comment:
Actually, I think `ensure_server_session` is a better name.  It is more explicit, and I don't think we should abbreviate method name.

Reply:
Done.

Reply:
How about `_materialize`? That's what ruby named their method and it conveys the meaning well.